.circleci/config.yml

```diff
@@ -231,7 +231,7 @@ jobs:
           command: sudo python3 -m pip install awscli
       - run:
           name: Deploy to S3
-          command: aws s3 sync generated_docs s3://purchases-docs/ios --delete --acl public-read
+          command: aws s3 sync generated_docs s3://purchases-docs/ios --delete
```

Comment:
In case anyone reading this is wondering, this is safe, since we're deploying to cloudfront anyway, and that's where we're hosting these files from.

Reply:
I've executed the sync and invalidation of the CloudFront manually with the circleci credentials and went fine without that flag